### PR TITLE
Add Hunan University with their CS faculty

### PR DESCRIPTION
**Pull request by actbee**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by actbee** [comment]
I have added Hunan University, China. Please check. Thanks.

File: csrankings.csv [hunk 1]
```diff
@@ -18164,3 +18164,36 @@ eva Kühn,TU Wien,http://www.complang.tuwien.ac.at/eva,NOSCHOLARPAGE
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Shizhe Zhou,Hunan University,https://shizhezhou.github.io/,NOSCHOLARPAGE
+Guo Chen,Hunan University,https://1989chenguo.github.io/,ouriG7cAAAAJ
+Zhe Quan,Hunan University,http://csee.hnu.edu.cn/people/quanzhe,NOSCHOLARPAGE
+Hao Chen0002,Hunan University,http://www.aimlab.org/haochen/,NOSCHOLARPAGE
+Xiangxiang Zeng,Hunan University,https://xzenglab.github.io/xzeng/,B20HBMIAAAAJ
+Kenli Li,Hunan University,http://csee.hnu.edu.cn/people/likenli,NOSCHOLARPAGE
+Di Wu0002,Hunan University,https://www.ics.uci.edu/~dwu3/,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv]: Missing space between the name and the suffix (this is one of several such cases).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (actbee) [reply csrankings.csv]: Ahhh, thank you for your reminding. I have fixed that problem. Please check again. Thank you.

**Comment by emeryberger** [comment]
Sorry, need to pull a new version and resolve conflicts (might want to wait till things settle down in a day or so).

**Review by emeryberger — APPROVED** [review]

**Comment by actbee** [comment]
Seems there is no conflicts now, please check again, Thanks!

**Comment by emeryberger** [comment]
Merged, thanks!

**Pull request merged**